config/checker/io.vavr.collection.astub

```diff
@@ -18,6 +19,10 @@ interface List<T> {
   @Pure @EnsuresMinLenIf(expression="this", result=false, targetValue=1) boolean isEmpty();
 }
 
+interface Map<K, V> {
+  @PolyNull V getOrElse(K key, @PolyNull V defaultValue);
```

Comment:
Needed for code around com/virtuslab/gitmachete/backend/root/GitMacheteRepositoryBuilder.java:93 to compile correctly:

```java
BaseGitMacheteBranch currentBranch = currentCoreBranch != null
        ? branchByName.getOrElse(currentCoreBranch.getName(), null)
        : null;
```

Reply:
Otherwise Checker would require `getOrElse` to accept as `defaultValue` & return the `V` type of the `Map`, i.e `@NonNull BaseGitMacheteBranch`... `@PolyNull` gives more flexibility in this respect